packages/react-core/src/components/Brand/examples/BrandBasic.tsx

```diff
@@ -0,0 +1,4 @@
+import React from 'react';
+import { Brand } from '@patternfly/react-core';
+
+export const BrandBasic: React.FunctionComponent = () => <Brand src="./pfLogo.svg" alt="Patternfly Logo" />;
```

Comment:
did it not work if you tried to import the svg? might be worth opening an issue if we have an issue with svg imports.

Reply:
We don't support importing svg files at the moment (probably a tsconfig setting, possibly webpack config related), but importing the SVG didn't seem correct to begin with because `src` takes a `string` type (implying a path to a source, not the source itself), not an `SVGElement` or `HTMLElement` type. 

I don't perceive it as an issue unless we need to import svg's somewhere. In this case we don't. Then there's the question of, do we really want to treat them as their own modules or do we just create function components that return JSX interpretations of the svg, which is another approach which wouldn't require any tsconfig/webpack changes.

Reply:
Looking into this further, the default import of an SVG element returns a string, so a module file was added to accommodate for typescript not recognizing SVG imports as modules (`assets.d.ts` - if we want to update the naming convention here, let me know), and this example was reverted back to using that imported `pfLogo`.